### PR TITLE
Simple HTTP Filter Example

### PR DESCRIPTION
**Pull request by mbertrone**

### CODE REVIEW

**Comment by 4ast** [comment]
looks good! I'm not sure why there are two examples: http-parse-v2 and http-parse. The latter one seems obsolete?

**Comment by mbertrone** [comment]
Thanks @4ast 
First version is the simple one: if the url is too long (splitted in more than one packet) is truncated. 
Second version is quite more complex: if necessary reassembles packets belonging to the same session and prints the complete url.

File: examples/networking/http_filter/http-parse-v2.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+  #
+  #IHL : Internet Header Length is the length of the internet header 
+  #value to multiply * 4 byte
+  #e.g. IHL = 5 ; IP Header Length = 5 * 4 byte = 20 byte
+  #
+  #Total Lenght: This 16-bit field defines the entire packet size, 
+  #including header and data, in bytes.
+
+  #calculate packet total lenght
+  total_lenght = packet_bytearray[ETH_HLEN + 2]               #load MSB
```
Comment (drzaeus77) [inline examples/networking/http_filter/http-parse-v2.py]: s/lenght/length/

**Comment by drzaeus77** [comment]
Can you remove the pdf files and instead send a pull request to https://github.com/iovisor/bpf-docs with those? Then the README can have a link to those files.

**Comment by drzaeus77** [comment]
@brendangregg Regarding why parse raw packets: I imagine this type of tool being used in a transit box, or perhaps where access to the original socket is not available. A hypervisor or linux based router for instance.

**Comment by mbertrone** [comment]
Some minor changes done as requested
Pdf documentation moved to bpf-docs
@brendangregg thanks for the advise, i used this approach mainly for the reason @drzaeus77 said

**Comment by 4ast** [comment]
"This branch is out-of-date with the base branch."
and cannot be merged. please rebase.

**Pull request merged**